.github/workflows/test-templates.yml

```diff
@@ -67,6 +67,12 @@ jobs:
       id-token: write
 
     steps:
+      - name: Free Disk Space (Ubuntu)
```

Comment:
The tests seem to frequently run out of disk space. We use this actions in many of our repos to help with this, and it seems to work nicely here.